src/validate/methods/arrayOfStrings.ts

```diff
@@ -0,0 +1,20 @@
+export const arrayOfStrings = (value: string) => {
+  if (typeof value !== "string") {
+    return "Expected strings of array.";
+  }
+
+  try {
+    const parsedValue = JSON.parse(value);
```

Comment:
Parsing an empty string will throw an error. We should add a check for that or update the catch generic message.